packages/orbit-components/src/Card/components/Header/index.tsx

```diff
@@ -81,11 +100,7 @@ const Header = ({
     {header && icon}
     {header}
     {expandable && !actions && <ChevronIcon color="secondary" expanded={expanded} />}
-    {actions && (
-      <Stack inline grow={false} justify="end">
-        {actions}
-      </Stack>
-    )}
+    {actions && <StyledActionsWrapper>{actions}</StyledActionsWrapper>}
```

Comment:
could not you achieve the same with the Stack as before?

Reply:
I tried keeping the Stack but I was unable to achieve. I don't think stacking two Stacks is a better approach than having one wrapper component. How would you suggest it?

Reply:
Managed to do it without the wrapper, thanks 👍